etcdctl/command/import_snap_command.go

```diff
@@ -85,14 +87,14 @@ func handleImportSnap(c *cli.Context) {
 	}
 	n := copyKeys(all.Node, setc)
 
-	hiddens := c.StringSlice("hidden")
-	for _, h := range hiddens {
-		allh, err := st.Get(h, true, true)
-		if err != nil {
-			handleError(ErrorFromEtcd, err)
-		}
-		n += copyKeys(allh.Node, setc)
-	}
+	//hiddens := c.StringSlice("hidden")
+	//for _, h := range hiddens {
+	//allh, err := st.Get(h, true, true)
+	//if err != nil {
+	//handleError(ErrorFromEtcd, err)
+	//}
+	//n += copyKeys(allh.Node, setc)
+	//}
```

Comment:
https://github.com/eparis/pflag/tree/SliceStrings

Has support for string slices. I can define it in the etcdctl code, but that'd suck, so i'd consider this a prereq before I remove the WIP....